libs/isograph-react/src/core/IsographEnvironment.ts

```diff
@@ -134,8 +137,11 @@ export function createIsographEnvironment(
   logFunction?.({
     kind: 'EnvironmentCreated',
   });
+  let optimisticLayer: OptimisticLayer = {};
```

Comment:
Rust brain has gotten to you
Also this should be caught by eslint?? (Also below)